Calculator/Calculator/Formula.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Formula.swift
+//  Calculator
+//
+//  Created by redmango1446 on 2023/06/06.
+//
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double> = CalculatorItemQueue()
+    var operatos: CalculatorItemQueue<Operator> = CalculatorItemQueue()
+    
+    mutating func result() throws -> Double {
+        var result: Double = 0
+        
+        guard !operatos.isEmpty else {
+            return operands.dequeue() ?? 0
+        }
+        
+        guard var lhs = operands.dequeue() else {
+            return 0
+        }
+        
+        while let currentOperator = operatos.dequeue() {
+            
+            guard let rhs = operands.dequeue() else {
+                throw CalculatorError.incompleteFormula
+            }
+            
+            result = try currentOperator.calculate(lhs: lhs, rhs: rhs)
+            lhs = result
```

Comment:
```suggestion
            lhs = try currentOperator.calculate(lhs: lhs, rhs: rhs)
```
result 변수를 사용하지 않고 lhs에 재할당하는 방법으로도 가능할 것 같습니다.
그렇다면 `return lhs`가 되고, lhs가 결과값이라는 것이 의미적으로 적절하지 않게 됩니다.
따라서 적절한 네이밍을 생각하면 좋을 것 같습니다.